{"name": "Проверка ошибки при попытке удаления курьера без указания id", "status": "passed", "steps": [{"name": "Удаление курьера", "status": "passed", "parameters": [{"name": "courier_id", "value": "''"}], "start": 1747228288413, "stop": 1747228288513}], "start": 1747228288413, "stop": 1747228288515, "uuid": "77f18dc6-929c-4846-a6d9-ef547bcb8646", "historyId": "0b46b44121eabc736bf51ec5ad9d7f19", "testCaseId": "0b46b44121eabc736bf51ec5ad9d7f19", "fullName": "tests.test_couriers.test_courier_delete.TestCourierDelete#test_courier_delete_error_with_empty_id", "labels": [{"name": "parentSuite", "value": "tests.test_couriers"}, {"name": "suite", "value": "test_courier_delete"}, {"name": "subSuite", "value": "TestCourierDelete"}, {"name": "host", "value": "DESKTOP-VE6DRBG"}, {"name": "thread", "value": "19428-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_couriers.test_courier_delete"}]}